### PR TITLE
Package clap.0.1.0

### PR DESCRIPTION
**Pull request by rbardou**
### `clap.0.1.0`
Command-Line Argument Parsing, imperative style with a consumption mechanism

---
* Homepage: https://github.com/rbardou/clap
* Source repo: git+https://github.com/rbardou/clap.git
* Bug tracker: https://github.com/rbardou/clap/issues

---
:camel: Pull-request generated by opam-publish v2.0.2

### CODE REVIEW

File: packages/clap/clap.0.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,16 @@
+opam-version: "2.0"
+maintainer: "Romain Bardou"
+authors: [ "Romain Bardou" ]
+homepage: "https://github.com/rbardou/clap"
+bug-reports: "https://github.com/rbardou/clap/issues"
+dev-repo: "git+https://github.com/rbardou/clap.git"
+license: "MIT"
```
Comment (mseri) [inline packages/clap/clap.0.1.0/opam:7]:
```suggestion
license: "MIT"

depends: [
   "ocaml" {>= "4.02"}
   "dune"
]
```

The package at least depends on dune. I would recommend to commit to the sources also your `dune-project` file

Comment (mseri) [reply packages/clap/clap.0.1.0/opam:7]: Please make these changes also upstream, I believe dune >= 1.0 should be fine for what I saw.
I have put ocaml 4.02 as dependency to check if a lower bound on the ocaml version is needed

Comment (rbardou) [reply packages/clap/clap.0.1.0/opam:7]: Thank you! I'll upstream the changes.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**